File: app_codebase/dart_bkend_base_02/bin/modules/service/service_app.dart
import 'package:shelf/shelf.dart';

import '../../core/http/contracts/http_response_contract.dart';
import '../../core/http/contracts/register_route_contract.dart';

void registerServiceRoutes(
  ApplicationRouteSink routes,
  HttpResponseContract res,
) {
  routes.serviceGet(
    '/health',
    (Request request) async =>
        res.jsonOk({'module': 'service', 'status': 'up'}),
  );
}

File: app_codebase/python_base_05/bin/modules/service/service_app.py
from core.http.contracts.register_route_contract import ApplicationRouteSink
from core.http.contracts.response_contract import HttpResponseContract


def register_service_routes(
    routes: ApplicationRouteSink,
    res: HttpResponseContract,
) -> None:
    routes.service_get(
        "/health",
        lambda: res.json_ok({"module": "service", "status": "up"}),
    )
